x-pack/test/case_api_integration/common/fixtures/plugins/cases_client_user/server/plugin.ts

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
This is a test plugin to make sure using the elasticsearch from `CoreStart` is ok. It also tests that using the plugin start contract works.